packages/contracts/contracts/L1/AssetManager.sol

```diff
@@ -315,6 +315,8 @@ contract AssetManager is ISemver, IERC721Receiver, IAssetManager {
         if (canWithdrawAt(validator) > block.timestamp) {
             revert NotElapsedMinDelegationPeriod();
         }
+        if (VALIDATOR_MANAGER.jailExpiresAt(validator) > block.timestamp)
```

Comment:
Could you explain why we need this here?

Reply:
Ah, I got the point so mark resolved.

Reply:
To prevent a validator in jail from immediately withdrawing and then registering with a new address to become active.